### PR TITLE
Disintegrate!

### PR DESCRIPTION
**Pull request by OCannings**
I noticed we weren't using these lines of code so I went ahead and
deleted them. Tests passing.

### CODE REVIEW

File: lib/juicebox.ex [hunk 1]
```diff
@@ -1,5 +1,4 @@
 defmodule Juicebox do
-  use Application
```
Comment (OCannings) [inline lib/juicebox.ex:2]: Probably don't need this

File: web/services/video_services.ex [hunk 2]
```diff
@@ -1,17 +1,10 @@
 defmodule Juicebox.VideoServices do
-  import Ecto.Query
 
-  alias Juicebox.Repo
   alias Juicebox.Video
 
   def find_or_create(%{video_id: video_id} = video) do
-    case Repo.get_by(Video, %{video_id: video_id}) do
```
Comment (OCannings) [inline web/services/video_services.ex:8]: this doesn't look important, bin it!

File: lib/juicebox.ex (hunk 1 above)
Comment (Gazler) [reply lib/juicebox.ex:2]: You should keep it in, it ensures your application first the application contract, it also allows you to stop the application. https://github.com/elixir-lang/elixir/blob/v1.3.3/lib/elixir/lib/application.ex#L149

**Comment by OCannings** [comment]
This pull request was tongue in cheek, please ignore.

**Comment by Gazler** [comment]
But I am 2 thirds the way through my review!

**Comment by ljkbennett** [comment]
2 thirds of the way through and only one comment. #winning

**Pull request closed without merging**